docs/datatypes/stream.md

```diff
@@ -22,6 +22,22 @@ import zio.stream._
 val streamFromIterable: Stream[Nothing, Int] = Stream.fromIterable(0 to 100)
 ```
 
+You might also want to create it from a callback API :
+```scala mdoc:silent
```

Comment:
This is not compiling, can you please run the docs task to see what the problem is? 

Thank you!